plenum/test/node_catchup/test_node_ledger_manager.py

```diff
@@ -1,9 +1,21 @@
-def test_ledger_sync_order(looper, txnPoolNodeSet):
+from plenum.common.constants import AUDIT_LEDGER_ID, POOL_LEDGER_ID
+from plenum.server.node import Node
+
+
+def test_ledger_sync_order(txnPoolNodeSet):
     for node in txnPoolNodeSet:
-        ledger_ids = node.ledger_ids
-        for idx, lid in enumerate(ledger_ids):
-            next_ledger_id = node.ledgerManager.ledger_to_sync_after(lid)
-            if idx != (len(ledger_ids) - 1):
-                assert next_ledger_id == ledger_ids[idx + 1]
-            else:
-                assert next_ledger_id is None
+        check_ledger_sync_order(node)
+
+
+def check_ledger_sync_order(node: Node):
+    leecher = node.ledgerManager._node_leecher
+
+    # TODO: It does ignore designated ledger order, however it will obsolete after implemnting parallel catchup
+    ledger_ids = [ledger.id for ledger in node.ledgerManager.ledgerRegistry.values()
```

Comment:
Why do we exclude Audit and Pool ledgers from checking?

Reply:
They are handled by separate states in NodeLeecherService, therefore they are not handled by get_next_ledger. Actually this function (and test) will be removed once parallel catch-up is implemented.